tests/unit/test_subprocpool.py

```diff
@@ -273,6 +273,20 @@ def test__run_command_exit_no_255_callback(caplog, mock_ctx):
     assert 'callback called' in caplog.records[0].msg
 
 
+def test__run_command_exit_no_gettable_platform(caplog, mock_ctx):
```

Comment:
I considered using monkeypatch to have `get_platform` to return both exceptions, but decided that was re-testing basic python Exception class inheritances.